### PR TITLE
Use client-side shuffle on "Commercial support" page

### PR DESCRIPTION
**Pull request by garbas**
Was #1052

### CODE REVIEW

File: js/nixos-site.js [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+    var seed = randomish(seedParam);
+
+    while (--len) {
+      let randIndex = Math.floor(seed * (len + 1));
+      [$find[randIndex], $find[len]] = [$find[len], $find[randIndex]];
+    }
+
+    $el.empty().append($find);
+  });
+};
```
Comment (pacien) [inline js/nixos-site.js:21]: (Repeating/rephrasing my comments from the previous pull request:)

For the Fisher-Yates shuffle to work, a new random number has to be used
each time an index is picked for swapping elements.

Also, the sinful function (used here to work around the lack of seedable PRNG
in the web browser) isn't fair.

How about dropping the hourly seed thing and just use `Math.random()` in the
`while` loop?

**Review by pacien — CHANGES_REQUESTED** [review]

File: js/nixos-site.js (hunk 1 above)
Comment (pacien) [reply js/nixos-site.js:21]: This: https://github.com/NixOS/nixos-homepage/pull/1052#discussion_r1193377878

Comment (drupol) [reply js/nixos-site.js:21]: I support this and I think that we should drop the hourly seed thingie.

File: Makefile [hunk 2]
```diff
@@ -255,7 +255,7 @@ index.html: blog/announcements-rss.xml blog/index.html
 community/commercial-support.html: community/commercial-support.html.in
 
 community/commercial-support.html.in: community/commercial-support.toml
```
Comment (jtojnar) [inline Makefile:257]: Copying my comment from https://github.com/NixOS/nixos-homepage/pull/1052#discussion_r1193009005:

> It should be possible to just have
> 
> ```suggestion
> community/commercial-support.html: community/commercial-support.toml
> ```
> 
> and remove the `.in` file from git.

File: js/nixos-site.js (hunk 1 above)
Comment (fricklerhandwerk) [reply js/nixos-site.js:21]: @drupol can you update this on your branch and reopen the original PR? Then we can merge something at all. I will hit the button if you ping me, let's fix forward at the worst.

Comment (drupol) [reply js/nixos-site.js:21]: I created a new PR @ https://github.com/NixOS/nixos-homepage/pull/1081

**Pull request closed without merging**